### PR TITLE
[cmake] Fix OpenCL enable print

### PR DESCRIPTION
**Pull request by adydychk**
Fixes https://github.com/Intel-Media-SDK/MediaSDK/issues/1014

### CODE REVIEW

**Comment by adydychk** [comment]
It is unclear from "enable" that these parameters not env specific but enabled by default or by parameter (so it is not final result).  
I tried to formulate it more clear if you have another suggestions - pls welcome :)

File: CMakeLists.txt [hunk 1]
```diff
@@ -173,7 +173,7 @@ endif()
 message("  CMAKE_INSTALL_FULL_INCLUDEDIR           : ${CMAKE_INSTALL_FULL_INCLUDEDIR}")
 message("  CMAKE_INSTALL_FULL_LIBDIR               : ${CMAKE_INSTALL_FULL_LIBDIR}")
 message("  CMAKE_INSTALL_FULL_DATADIR              : ${CMAKE_INSTALL_FULL_DATADIR}")
-message("Enable:")
+message("Enabled project configuration (by default or parameters):")
```
Comment (dvrogozh) [inline CMakeLists.txt]: I personally don't understand what you mean under this phrase. To me it is misleading. The previous "Enable" is honest and straightforward: here is list of variables and here is their values. Just read this literally. This is not indication whether feature is really enabled or not, that's list of variables. The fact that OCL could be OFF while variable is ON - that's a bug. We need to fix it, not workaround it.

File: CMakeLists.txt [hunk 2]
```diff
@@ -174,7 +174,9 @@ message("  CMAKE_INSTALL_FULL_INCLUDEDIR           : ${CMAKE_INSTALL_FULL_INCLUD
 message("  CMAKE_INSTALL_FULL_LIBDIR               : ${CMAKE_INSTALL_FULL_LIBDIR}")
 message("  CMAKE_INSTALL_FULL_DATADIR              : ${CMAKE_INSTALL_FULL_DATADIR}")
 message("Enable:")
-message("  ENABLE_OPENCL                           : ${ENABLE_OPENCL}")
+if(DEFINED OPENCL_FOUND)
+  message("  ENABLE_OPENCL                           : ${ENABLE_OPENCL}")
+endif()
```
Comment (adydychk) [inline CMakeLists.txt]: @dvrogozh 
I hope that it is better...

I tried to "play" with something as:
```
option( ENABLE_OPENCL "Build targets dependent on OpenCL?" FALSE )
                                                                                                           ^
```
But it is all time defined as TRUE... I don\`t know why possible I did it wrong...

Comment (adydychk) [reply CMakeLists.txt]: Or I can implement:
```
if(DEFINED OPENCL_FOUND)
  message("  ENABLE_OPENCL                           : ${ENABLE_OPENCL}")
else()
  message("  ENABLE_OPENCL                           : OFF")
endif()
```

Output:
```
Enable:
  ENABLE_OPENCL                           : OFF
  ENABLE_X11_DRI3                         : OFF
  ENABLE_WAYLAND                          : OFF
  ENABLE_ITT                              : OFF
  ENABLE_TEXTLOG                          : OFF
  ENABLE_STAT                             : OFF
Build:
...
```

Comment (adydychk) [reply CMakeLists.txt]: @onabiull Your opinion on this https://github.com/Intel-Media-SDK/MediaSDK/pull/1022#discussion_r245482645? Is it ready to merge? Or should I to re-implement as in https://github.com/Intel-Media-SDK/MediaSDK/pull/1022#discussion_r245483252 ?

**Pull request closed without merging**